### PR TITLE
Missing Covers | Added [part 2]

### PR DESCRIPTION
**Pull request by andercard0**
The following covers has been added:

SCES-00269
SCUS-94480
SCUS-94480-1
SCUS-94702
SLES-00156
SLES-00543
SLES-00749
SLES-03298
SLES-03983
SLES-13158
SLES-13493
SLPM-86621
SLPS-00140
SLPS-00216
SLPS-00681
SLPS-00817
SLPS-00833
SLPS-00899
SLPS-00944
SLPS-01219
SLPS-01692
SLPS-01831
SLPS-02021
SLPS-02034
SLPS-02038
SLPS-02239
SLPS-02274
SLPS-02374
SLPS-02805
SLPS-03577
SLUS-00175
SLUS-00176
SLUS-00718
SLUS-00752
SLUS-00899
SLUS-00908
SLUS-01164
SLUS-01239
SLUS-01252
SLUS-01367

For people who is going to use this please, consider to double check if the covers the program tells you as missing are in fact missing. We have 2 scenarios here, First, you have a good piece of software here that fetches all of your covers and download it for you automatically, second, being that easy, is not perfect. Why? well, some games in Playstation era had 2 or more discs meaning you had a box with a 3 or more discs with the same cover :p

In conclusion, if some of your art/cover is missing double check if it is not using the same id number for that game, if so, it is not the software fault! :) Download the cover for yourself.

should be enough for: #3 #5 #2

### CODE REVIEW

**Comment by LouiseSulyvahn** [comment]
Thanks for letting me know about multi-disc CDs, I wasn't sure whether to use the cover from CD 1 or add some kind of watermark to indicate it's another CD.

But I already decided to add the main cover.

**Comment by andercard0** [comment]
> Thanks for letting me know about multi-disc CDs, I wasn't sure whether to use the cover from CD 1 or add some kind of watermark to indicate it's another CD.
> 
> But I already decided to add the main cover.

All good. Here, there is a example: SCUS-94480-1 this number 1 means that this cover is meant for the Disc 1 the other discs may or may not adopt the same patern. Making the script bot be so precise when it comes to march all user enviroments.

That said I would suggest, if you allow me.. to make it a bit more clear to end user in regards of it; Warning that "if you have a multi-disc game, please consider to download other covers manually". 

Or something of that sort would work alreadt. 😁
